lib/PlaidClient.js

```diff
@@ -477,17 +477,17 @@ Client.prototype.listPaymentRecipients =
     }, cb);
   };
 
-// createPayment(String, String, Object, Function)
+// createPayment(String, String, Object, Object, Function)
```

Comment:
nit: to be consistent with the comments above, I think this should be `Object?` so it's clear that the value is optional

Reply:
updated.